src/lib/core/CHIPError.h

```diff
@@ -2400,6 +2400,15 @@ using CHIP_ERROR = ::chip::ChipError;
  */
 #define CHIP_ERROR_INVALID_FILE_IDENTIFIER                     CHIP_CORE_ERROR(0xda)
 
+/**
+ * @def CHIP_ERROR_BUSY
+ *
+ * @brief
+ *   The Resource is busy and cannot process the request. Trying again might work.
+ */
+#define CHIP_ERROR_BUSY                     CHIP_CORE_ERROR(0xdb)
```

Comment:
I'm surprised restyle doesn't catch this!

Reply:
All the other error codes seem to be following a similar spacing paradigm...

Reply:
It seems like all the other `CHIP_CORE_ERROR` line up in the file...

Reply:
All the error codes seem to line up, until CHIP_ERROR_IM_MALFORMED_COMMAND_STATUS_IB (line 2233). From there on it gets wonky.